.github/ISSUE_TEMPLATE/api_request.yml

```diff
@@ -0,0 +1,25 @@
+name: API Request
+description: Create an API request for Quilt Standard Libraries.
+labels: [enhancement]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking your time to create your API request! If you want to discuss your feature request, join us on [Discord](https://discord.quiltmc.org).
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe how your feature(s) will work if applicable.
+      description: "Describe what you want QSL to implement as clearly as possible. More info means your request is more likely to be accepted."
```

Comment:
```suggestion
  - type: textarea
    id: description
    attributes:
      label: Describe what API(s) you want. 
      description: "Describe what API you believe QSL should implement. What does the API do? Is there a similar implementation, and if so how should QSL differ from it? The more detailed your request is the easier it will be for people to begin implementation of the API."
```